src/DataManagement/Entries/EntryList.cs

```diff
@@ -8,12 +10,58 @@ public class EntryList : List<IEntry>, IEntryList
     {
         //I added EntryType to EntryList so we don't have to pass an EntryType as a parameter in methods as often
         public EntryType EntryType { get; set; }
-
         public EntryList(EntryType entryType)
         {
             EntryType = entryType;
         }
 
+        private EntryList(EntryType entryType, IEnumerable<IEntry> entryList)
+        {
+            EntryType = entryType;
+            if (entryList is null) return;
+            AddRange(entryList);
+        }
+        
+        public IEntryList GetBy(Importance importance)
+        {
+            return new EntryList(EntryType, this.Where(x => x.Importance == (int)importance).ToList());
+        }
+
```

Comment:
Are all these methods avaible for Income/Expenses or for the Goals as well, not necessarily that we would use it for goals, just interesting.

Reply:
I've just added to Incomes and Expenses for now at least